src/resources/Tailgate/Tailgate.ts

```diff
@@ -0,0 +1,22 @@
+import API from '../../APICore.js';
```

Comment:
We don't really want to expose this

Reply:
I read the other class and they has the placeholder, in our case without the placeholder what should we expect to use for organizationid in tailgate getWorkerCount call?